warehouse/integrations/bigquery/bigquery_test.go

```diff
@@ -1042,3 +1057,14 @@ func TestUnsupportedCredentials(t *testing.T) {
 	assert.NotNil(t, err)
 	assert.Contains(t, err.Error(), "client_credentials.json file is not supported")
 }
+
+func cloneWarehouse(t *testing.T, wh model.Warehouse) model.Warehouse {
+	t.Helper()
+	buf, err := json.Marshal(wh)
+	require.NoError(t, err)
+
+	var clone model.Warehouse
+	require.NoError(t, json.Unmarshal(buf, &clone))
+
+	return clone
+}
```

Comment:
Should we create a helper for this, since it's being used at couple of places?

```GO
func Clone[T](t *testing.T, s T) T {
	t.Helper()

	buf, err := json.Marshal(s)
	require.NoError(t, err)

	var clone T
	require.NoError(t, json.Unmarshal(buf, &clone))

	return clone
}
```

Reply:
Done :+1: